test/live_select_tags_test.exs

```diff
@@ -345,7 +364,10 @@ defmodule LiveSelectTagsTest do
 
     send_update(live, value: [3, 5], options: [{"C", 3}, {"D", 4}, {"E", 5}])
 
-    assert_selected_multiple(live, [%{label: "C", value: 3}, %{label: "E", value: 5}])
+    assert_selected_multiple(live, [
+      %{label: "C", value: 3, sticky: false},
+      %{label: "E", value: 5, sticky: false}
+    ])
```

Comment:
don't need sticky here. We just need to assert options are selected, we don't care if they're sticky or not